src/Components/CreateImageWizard/utilities/requestMapper.ts

```diff
@@ -377,20 +381,28 @@ export const mapRequestToState = (request: BlueprintResponse): wizardState => {
     },
     registration: {
       registrationType:
-        request.customizations?.subscription && isRhel(request.distribution)
-          ? request.customizations.subscription.rhc
-            ? 'register-now-rhc'
-            : 'register-now-insights'
-          : getSatelliteCommand(request.customizations.files)
-          ? 'register-satellite'
-          : 'register-later',
+        aapRegistration?.aap_registration
```

Comment:
This is wrong and should be rewritten with a helper function that uses a switch for readability.